e2eAIOK/DeNas/search.py

```diff
@@ -56,8 +57,12 @@ def main(params):
     elif params.domain == 'asr':
         with open(params.supernet_cfg) as f:
             cfg = edict(yaml.safe_load(f))
-        super_net = TransformerASRSuper
-        search_space = {'num_heads': cfg.SEARCH_SPACE.NUM_HEADS, 'mlp_ratio': cfg.SEARCH_SPACE.MLP_RATIO,
+        if "pruner" in params and params.pruner.pruner:
```

Comment:
what's the 2nd parameter?

Reply:
we grouped pruner config in params.pruner, and params.pruner.pruner is to control whether we enable pruner